jena-fuseki2/apache-jena-fuseki/log4j2.properties

```diff
@@ -47,8 +47,8 @@ logger.jetty.name  = org.eclipse.jetty
 logger.jetty.level = WARN
 
 # Hide bug in Shiro 1.5.0
-logger.shiro.name = org.apache.shiro.realm.text.IniRealm
-logger.shiro.level = ERROR
```

Comment:
Fix logging config mistake - should set for "org.apache.shiro" (silences two log lines as the server starts).